src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -67,7 +67,11 @@ public class DefaultWatch3DViewModel : NotificationObject, IWatch3DViewModel
         protected List<NodeModel> recentlyAddedNodes = new List<NodeModel>();
         protected bool active;
         protected bool isGridVisible;
-        private readonly List<IRenderPackage> currentTaggedPackages = new List<IRenderPackage>();
+
+        /// <summary>
+        /// Represents the name of current Watch3DViewModel which will be saved in preference settings
+        /// </summary>
+        public virtual string PreferenceWatchName { get { return "IsBackgroundPreviewActive"; } }
```

Comment:
This is API breaking change. Dynamo Revit will no longer work on Dynamo Core 1.0